browser/ntp_background/ntp_background_prefs.cc

```diff
@@ -62,8 +62,7 @@ void NTPBackgroundPrefs::RegisterPref(
   dict.Set(kSelectedValueKey, "");
   registry->RegisterDictionaryPref(kPrefName, base::Value(std::move(dict)));
 
-  registry->RegisterListPref(kCustomImageListPrefName,
-                             base::Value(base::Value::Type::LIST));
+  registry->RegisterListPref(kCustomImageListPrefName, base::Value::List());
```

Comment:
minor: We can omit the empty default value because there are overloads that do the same thing.

Reply:
Ah, just checked the codebase... I didn't know that. Thanks for pointing that out.